src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,97 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elements = new Object[DEFAULT_CAPACITY];
+    private int size;
+
     @Override
     public void add(T value) {
-
+        if (elements.length == size) {
+            Object[] newElements = new Object[elements.length * 3 / 2 + 1];
+            System.arraycopy(elements, 0, newElements, 0, size);
+            elements = newElements;
```

Comment:
lets create a separate method to grow a table

Reply:
I think Nastya meant moving this 26,27,28 lines to a method `grow()`, like this
```
if (elements.length == size) {
    grow();
}
```
but you can leave this `ensureCapacity()` method